helper.go

```diff
@@ -146,3 +138,10 @@ func getRequestURL(w http.ResponseWriter, r *http.Request) *url.URL {
 
 	return URL
 }
+
+func serveLandingPage(w http.ResponseWriter, r *http.Request)  {
+	b, _ := ioutil.ReadFile("public/index.html")
+	w.WriteHeader(http.StatusOK)
+	_, _ = w.Write(b)
+	return
```

Comment:
Set Content-type header to `text/html`
```suggestion
	w.Header().Set("Content-Type", "text/html; charset=utf-8")
	return
```